tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +114,73 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableOrchestratorModule() error {
```

Comment:
Instead of the test suite enabling this module here, how about we set the following in the test cluster CR?
```yaml
  mgr:
    modules:
      - name: rook
        enabled: true
```

#5834 Disabled the rook module by default, but the operator will still enable it and set the orchestrator backend to it if this is specified.

Reply:
I think it is already set:
https://github.com/rook/rook/blob/ab728e0183c92e059af7d663b287b00e95d6e175/tests/framework/installer/ceph_manifests.go#L227

In any case to have the rook mgr module verification in the test is always good, the test will only be executed if the module is enabled and active. if by any issue the rook module is not ready the test itself tries to enable it, and we will "bypass" the problem in this kind of situation.

Reply:
Oh if it's already set in the test cephcluster CR, why do we need to set it again in the test? Sounds good to have the test do verification of the module being enabled, but the test shouldn't have to also enable it.

Reply:
I have included your suggestion about warning if the orchestrator backend is not rook. IIRC we include the automatic enable to avoid the test failing ( i think it was a moment were the rook orchestrator was not enable sometimes... and we weren't able to find out the cause)

I can remove the automatic enable of the rook orchestrator if not set ,... but i think that the test is more robust now.. and the enable only happens when is really needed.